release-highlights/release-highlights-2022.07/release-highlights.md

```diff
@@ -0,0 +1,37 @@
+# PatternFly Release Highlights
+## Release 2022.07
+----------------------------------------------------------
+## New features and enhancements
+
+### [Allow expandable section toggle to be an arbitrary component](https://staging.patternfly.org/v4/components/expandable-section#with-custom-toggle-content)
```

Comment:
Can we update this URL to the live site rather than staging?
```suggestion
### [Allow expandable section toggle to be an arbitrary component](https://www.patternfly.org/v4/components/expandable-section#with-custom-toggle-content)
```